openstf/model/metamodels/missing_values_handler.py

```diff
@@ -0,0 +1,120 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+import numpy as np
+import pandas as pd
+from sklearn.base import BaseEstimator, RegressorMixin, MetaEstimatorMixin, clone
+from sklearn.pipeline import Pipeline
+from sklearn.impute import SimpleImputer
+from sklearn.utils.validation import check_is_fitted, check_X_y, check_array
+
+
+class MissingValueHandler(BaseEstimator, RegressorMixin, MetaEstimatorMixin):
+    """Class for a meta-models that handles missing values and removes columns filled exclusively by NaN.
+    Pipeline of imputer for missing value and a regressor.
+
+    Parameters
+    ----------
+
+    base_estimator: RegressorMixin
+        Regressor used in the pipeline.
+
+    missing_values: default=np.nan
+        Type of missing values handled by the imputer.
+
+    imputation_strategy: str, default=None
+        Imputing strategy for the imputer. If None, nan values are not allow and the linear regression will raise a ValueError.
+
+    fill_value: optional
+        Value used by the imputer for missing values.
```

Comment:
Could you add what sensible options are? Are we looking for a specific float, or a strategy (e.g. 'mean')

Reply:
This parameter is useful when the filling stratgey is to replace by a user defined constant value. The docstring should be clearer indeed.

Reply:
done.